3cn-ecn/nantralPlatform
# React app filter translation datepicker

# Description

- Translation of the FilterBar
- Date Picker improvement, including a clear button

# Tests

Go to page Event : 
- change the language in the user menu : the 'Filtres' button and the whole FilterBar is translated
- open the 'Date' Filter : if you click on the calendar icon, it opens a date picker. You can also use your keybord to type a date. When there is a date in the picker, a clear button appears. If you click on it, it clears the picker.

# Screenshots

French :
![image](https://user-images.githubusercontent.com/112811926/221527597-c623d7f7-4402-4773-9403-61c82d1519f8.png)
![image](https://user-images.githubusercontent.com/112811926/221527670-0068c2d9-d04e-4235-8e87-61412637320c.png)
English : 
![image](https://user-images.githubusercontent.com/112811926/221527800-113de1b4-e5e6-4aa3-827c-aad9cb77f402.png)
![image](https://user-images.githubusercontent.com/112811926/221527882-d0bbcaf1-1825-4131-96b6-bd4ed1d9a6e0.png)

![image](https://user-images.githubusercontent.com/112811926/221528025-656810d6-c3b8-423d-86bb-478cb96db3c6.png)
![image](https://user-images.githubusercontent.com/112811926/221528064-ece64664-7573-481c-aa50-39f113f1e870.png)
